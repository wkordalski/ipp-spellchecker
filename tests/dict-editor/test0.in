insert a
hints b
insert c
hints b
save dict.txt
clear
hints b
load dict.txt
hints b
