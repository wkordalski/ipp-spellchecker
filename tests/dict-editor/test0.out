inserted: a
a
inserted: c
a c
dictionary saved in file dict.txt
cleared

dictionary loaded from file dict.txt
a c
